src/components/FeedbackForm/index.jsx

```diff
@@ -67,6 +67,16 @@ export const FeedbackForm = ({
                 if (isFunction(onVote)) {
                     onVote(id);
                 }
+
+                if (typeof plausible === 'function') {
```

Comment:
I think this should be in the `onVote` function wherever it's used - just to keep the component away from a script that can potentially not exist.

However, a change would still be needed so that onVote also passed the code and the resourceId, maybe as a second param object that contains "extra" data